pymc3/theanof.py

```diff
@@ -38,11 +42,16 @@ def cont_inputs(f):
     return typefilter(inputvars(f), continuous_types)
 
 
+def floatX(X):
+    return np.asarray(X, dtype=theano.config.floatX)
+
+floatX_str = theano.config.floatX
```

Comment:
Why do you use this shortcut? It becomes a constant, not reference. So if anyone changes floatX, then this constant will not inherent changes. It can cause unexpected errors.